src/components/Search.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
```

Comment:
The search bar and clear input functionality looks very slick. Nice job!

Reply:
Agree! This looks great!